src/main/java/com/company/sample/application/SalesSystem.java

```diff
@@ -50,10 +50,9 @@ public static void main(String[] args) {
         //Start list order thread
         ListOrderThread listOrderThread = new ListOrderThread();
 
-        DateFormat currentDateFormat = new SimpleDateFormat("dd MMM yyyy");
+        DateFormat currentDateFormat = new SimpleDateFormat("yyyy/MM/dd");
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer

The SimpleDateFormat object was created without setting the time zone. Make sure that you want to use the default time zone or use letter z, Z or X in the pattern. Otherwise, if you don?t call setTimeZone() on the created SimpleDateFormat object, you might get an unexpected date and time when using the object.